### ISSUE TITLE
[Feature request] Coop area

### ISSUE THREAD

**Issue by sam01101** [BentoBoxWorld/BSkyBlock#422]
#### Is your feature request related to a problem?

Sometimes when player have coop permission, they can edit owner's whole world, that can cause
owner's valuable item like spawners etc

#### Describe the solution you'd like us to implement.

Coop area can let owner to limit the cooper which area they are allowed to break/create.

#### Describe alternatives you've considered.

This is the origin alternative that I've think of

### Additional context

- Some temporary tools to plot out which area (Like WorldEdit's axe)

<details>
<summary>Out Topic comment</summary>

I got this idea by sudden and this's a great idea to against grifter and stealer
</details>

**Comment by tastybento** [comment]
This is an interesting idea. I'm not sure how it'd work yet, but it's interesting.

**Comment by user670** [comment]
Does that mean there will be two sets of permissions, one for regular areas, one for co-op areas? Say, break block permission is owner for regular areas and trusted-or-above for co-op areas

(does that overcomplicate stuff?)

**Comment by sam01101** [comment]
> Does that mean there will be two sets of permissions, one for regular areas, one for co-op areas? Say, break block permission is owner for regular areas and trusted-or-above for co-op areas
> 
> (does that overcomplicate stuff?)

No, it’s just limiting the cooper where they have permission to edit
